backend/app/views/questions/show.slim

```diff
@@ -1,6 +1,13 @@
 h1 = @question.name
-h5 = "This is a #{@question.question_type.name.downcase} question."
+
+- if @question.single_choice?
```

Comment:
I believe it can be done with something like this, so we won't need if and else
``` 
 h5 = t("questions.show.#{@question.question_type.name})."
```

Reply:
@edimossilva
Okay I will refactor it!

Reply:
```
 h5 = t("questions.show.#{@question.question_type.name.downcase.join("_")})."
```

Reply:
nice job